src/components/interaction-designer/clipboard/prompt-kinds/Message.vue

```diff
@@ -26,26 +28,46 @@ export default {
   },
   props: {
     context: IContext,
-    prompt: MessagePrompt,
+    index: Number,
```

Comment:
Can you pls add a `//TODO` here to remind us avoiding repeating codes when we transform this into `vue-class` based component? eg:
> `//TODO: use class inheritance to avoid repeating codes when we transform this into `vue-class` based component

Reply:
Added!